Automattic/newspack-plugin
# feat(reader-activation): customizable UI text for auth modal

### All Submissions:






### Changes proposed in this Pull Request:



Adds customizable text to the auth UI, which can serve as a place for the publisher to specify additional details about the authorization. 

| Engagement wizard | front-end (one of 3) | 
|--------------|-----------|
| <img width="920" alt="image" src="https://github.com/user-attachments/assets/03b4115b-4183-4bc8-b7f4-9551eb774692"> | <img width="577" alt="image" src="https://github.com/user-attachments/assets/434c26e2-09ea-4f6b-ad13-0453b505528c"> |


#### To Do

- labels in the Engagement wizard
- polishing the UI in the Engagement wizard

### How to test the changes in this Pull Request:

1. View the auth modal UI, confirm nothing has changed
2. Open the Engagement wizard and observe a new section in the Advanced Settings: "Sign-in UI text"
3. Fill in the UI text, using text formatting options available
4. On the front-end, confirm the changes are reflected

### Other information:




---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208754020926479
